src/OctoshiftCLI.Tests/GithubClientTests.cs

```diff
@@ -53,6 +53,28 @@ public async Task GetAsync_Returns_String_Response()
             actualContent.Should().Be(EXPECTED_RESPONSE_CONTENT);
         }
 
+        [Fact]
+        public async Task Custom_Headers_Are_Added_And_Removed()
+        {
+            // Arrange
+            const string ghraphQLSchemaHeaderName = "GraphQL-schema";
+            const string ghraphQLSchemaHeaderValue = "internal";
+            var handlerMock = MockHttpHandlerForGet();
+            using var httpClient = new HttpClient(handlerMock.Object);
+            var githubClient = new GithubClient(_mockOctoLogger.Object, httpClient, null, null);
+
+            // Act
+            var internalSchemaHeader = new Dictionary<string, string>() { { ghraphQLSchemaHeaderName, ghraphQLSchemaHeaderValue } };
+            await githubClient.GetAsync("http://example.com", internalSchemaHeader);
+
+            // Assert
+            handlerMock.Protected().Verify(
+                "SendAsync",
+                Times.Once(),
+                ItExpr.Is<HttpRequestMessage>(msg => msg.Headers.Any(kv => kv.Key == ghraphQLSchemaHeaderName && kv.Value.First() == ghraphQLSchemaHeaderValue)),
```

Comment:
```suggestion
            const string graphQLSchemaHeaderName = "GraphQL-schema";
            const string graphQLSchemaHeaderValue = "internal";
            var handlerMock = MockHttpHandlerForGet();
            using var httpClient = new HttpClient(handlerMock.Object);
            var githubClient = new GithubClient(_mockOctoLogger.Object, httpClient, null, null);

            // Act
            var internalSchemaHeader = new Dictionary<string, string>() { { graphQLSchemaHeaderName, graphQLSchemaHeaderValue } };
            await githubClient.GetAsync("http://example.com", internalSchemaHeader);

            // Assert
            handlerMock.Protected().Verify(
                "SendAsync",
                Times.Once(),
                ItExpr.Is<HttpRequestMessage>(msg => msg.Headers.Any(kv => kv.Key == graphQLSchemaHeaderName && kv.Value.First() == graphQLSchemaHeaderValue)),
```

if we don't fix this typo it will drive me insane!